utils/gyb.py

```diff
@@ -1042,18 +1042,19 @@ def execute_template(
     ... THIS SHOULD NOT APPEAR IN THE OUTPUT
     ... ''')
     >>> out = execute_template(ast, line_directive='//#sourceLocation', x=1)
-    >>> out = out.replace(os.path.abspath(os.sep) + 'dummy.file', "DUMMY-FILE")
+    >>> working_directory = os.getcwd() + os.sep
+    >>> out = out.replace(working_directory, "WORKING-DIRECTORY/")
```

Comment:
Thanks for taking a look Dave. Could you clarify this comment? Basically, the issue here is that `/dummy.file` is transformed to `C:/dummy.file` on Windows (it stays the same on Unix/macos)

The old solution basically used `os.path.abspath(os.sep)` to evaluate the root path in a platform independent way:
- On Windows: `os.path.abspath(os.sep)` == 'C:/'`
- On Unix/macos: `os.path.abspath(os.sep) == '/'`

Then, replacing the occurence of `os.path.abspath(os.sep)/dummy.file` gets the correct result.

I'm really not sure a better way to test for this behaviour!